### ISSUE TITLE
Loss does not converge

### ISSUE THREAD

**Issue by sangjun04** [XPixelGroup/HAT#110]
First of all, thank you so much for a great work. really appreciate it.

I am trying to fine tune the train_HAT-L_SRx4_finetune_from_ImageNet_pretrain model with a custom data,but the loss is not converging at all.

I am using the default setting,

**# general settings
name: train_HAT-L_SRx4_finetune_from_ImageNet_pretrain
model_type: HATModel
scale: 4
num_gpu: auto
manual_seed: 0

# dataset and data loader settings
datasets:
  train:
    name: Custom_data
    type: PairedImageDataset
    dataroot_gt: ../train/hr
    dataroot_lq: ../train/lr_deblurred_130000
    meta_info_file: meta_info_custom.txt
    io_backend:
      type: disk**

and trained for over 100000 iterations, and the results are getting worse.

I am 100 percent sure there is nothing wrong with my custom dataset, since I used the same dataset to train SwinIR and the train loss converged well.

Any idea what might have gone wrong?

**Comment by abdullahbas** [comment]
Same here. Fine tuned model is worse than the pretrained one. I think either something wrong with the source code or they used different loss function.
